pkg/bee/cluster.go

```diff
@@ -287,6 +344,24 @@ func (c *Cluster) Settlements(ctx context.Context) (settlements ClusterSettlemen
 	return
 }
 
+// FlattenSettlements returns aggregated NodeGroupSettlements
+func (c *Cluster) FlattenSettlements(ctx context.Context) (settlements NodeGroupSettlements, err error) {
+	s, err := c.Settlements(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	settlements = make(NodeGroupSettlements)
+
+	for _, v := range s {
+		for n, set := range v {
+			settlements[n] = set
```

Comment:
for all of these flatten methods, i would return an error in case a node name was overridden, since this could lead to some inconsistent state in the test